core/src/main/java/io/grpc/Deadline.java

```diff
@@ -139,4 +139,9 @@ public long timeRemaining(TimeUnit unit) {
     Preconditions.checkNotNull(scheduler, "scheduler");
     return scheduler.schedule(task, deadlineNanos - System.nanoTime(), TimeUnit.NANOSECONDS);
   }
+
+  @Override
+  public String toString() {
+    return String.valueOf(deadlineNanos);
```

Comment:
FWIW I just implemented a timer heap in a personal project of mine, and I agree with Eric: relative times are really useful, absolute times are harder to read.  Since log statements typically contain a timestamp anyways, relative doesn't seem so bad.